webcompat/static/css/src/label-editor.css

```diff
@@ -70,8 +85,15 @@
 
 /* checkbox */
 .label-editor-list-item-checkbox {
-  margin: 0;
-  padding: 0;
+  display: none;
+}
+
+.label-editor-list-item-checkbox:checked + .label-editor-list-item::before {
+  content: "︎☑︎";
```

Comment:
Something I'm not clear about is why do we need to add a content.

ah… because we can't reorder the parent… in CSS.
and no luck there. https://caniuse.com/#feat=css-has This is the right use case for this.

Reply:
Yes, luckily label and input are connected through the `id` and `for` attributes. So we don't rely on wrapping the label around the input. And CSS order does only a visual reordering, not a structural one.